### DISCUSSION TITLE
Create new notification event?

### DISCUSSION THREAD

**Discussion by ksclark** [zwave-js/zwave-js#4238] [Ideas]
Is it possible to create a new notification event to be sent out for the web socket?
The specific example I have is:

I have a Domitec dimming light bulb that is in a desk lamp that has a manual switch to turn on / off.

When the lamp is turned on, I see the following in the `zwavejs` logs .
```
today at 7:54:53 AM2022-02-15T12:54:53.981Z SERIAL « 0x0114004984040e0411015e86725a8559732627707a6c                      (22 bytes)
today at 7:54:53 AM2022-02-15T12:54:53.983Z SERIAL » [ACK]                                                                   (0x06)
today at 7:54:53 AM2022-02-15T12:54:53.984Z DRIVER « [Node 004] [REQ] [ApplicationUpdateRequest]
today at 7:54:53 AM                                    payload: 0x040e0411015e86725a8559732627707a
today at 7:54:53 AM2022-02-15T12:54:53.987Z CNTRLR « [Node 004] Received updated node info
```

However, that does not trigger a notification message for websocket.  I would like to get that triggered as a notification such as `value updated` . In the openzwave implementation that triggers the `SIGNAL_VALUE` message . 

It is also possible that I am overlooking something.

**Comment by AlCalzone** [comment]
This is not supposed to update a value. Some older devices use this message to tell the driver to query it again. Not sure this is happening, since the log is truncated.

Please re-interview the device, so I can see all relevant information about it, then toggle it again manually. Afterwards, post the logfile here.

**Reply by ksclark** [reply]
Ah gotcha.  Excuse my ignorance, but is there a way to re-interview via a websocket command? Or is there another preferred way to do that .

**Reply by AlCalzone** [reply]
Depends on how you use zwave-js. Both the official HomeAssitant addon and zwavejs2mqtt offer this as an action in the UI.

**Reply by ksclark** [reply]
Attached a text file with the log contents from the `re-interview` 

[node_interview.txt](https://github.com/zwave-js/node-zwave-js/files/8072921/node_interview.txt)

After the interview, I then manually turned the lamp off and back on.  The logs then showed.

```
today at 11:16:40 AM2022-02-15T16:16:40.219Z SERIAL « 0x0114004984040e0411015e86725a8559732627707a6c                      (22 bytes)
today at 11:16:40 AM2022-02-15T16:16:40.220Z SERIAL » [ACK]                                                                   (0x06)
today at 11:16:40 AM2022-02-15T16:16:40.221Z DRIVER « [Node 004] [REQ] [ApplicationUpdateRequest]
today at 11:16:40 AM                                    payload: 0x040e0411015e86725a8559732627707a
today at 11:16:40 AM2022-02-15T16:16:40.222Z CNTRLR « [Node 004] Received updated node info
```

**Reply by AlCalzone** [reply]
I've raised https://github.com/zwave-js/node-zwave-js/issues/4241 for this.
The device supports Z-Wave+, which is currently our trigger to not refresh values when this message is received. I haven't heard of any other device with Z-Wave+ support that does not send Multilevel Switch reports via the lifeline, but this one apparently was programmed this way. This is okay per the specs, but does not make any sense IMO.
